csv2wiki/__init__.py

```diff
@@ -693,19 +707,6 @@ def _wiki_escape_page_title(self, s):
             s = s[:-1]
         return s
 
-    def _format_cell(self, cell):
```

Comment:
I agree with the name change (and doc updates) of this function -- thank you.

Reply:
And when I say "function", I mean "method".

OOP: It's what's for dinner.

Reply:
:cow: